packages/api/tests/lambdas/test-sqs-message-consumer.js

```diff
@@ -362,3 +365,85 @@ test.serial('SQS message consumer queues correct number of workflows for rules m
     await cleanupRulesAndQueues(rules, [queue]);
   });
 });
+
+test.serial('processQueues archives messages from the ENABLED sqs rule only', async (t) => {
+  const { stackName } = process.env;
+  const { rules, queues } = await createRulesAndQueues();
+  const message = { testdata: randomString() };
+
+  // Send message to ENABLED queue
+  const firstMessage = await SQS.sendSQSMessage(
+    queues[0].queueUrl,
+    message
+  );
+
+  // Send message to DISABLED queue
+  const secondMessage = await SQS.sendSQSMessage(
+    queues[1].queueUrl,
+    { testdata: randomString() }
+  );
+  const enabledQueueKey = getS3KeyForArchivedMessage(stackName, firstMessage.MessageId);
+  const deadLetterKey = getS3KeyForArchivedMessage(stackName, secondMessage.MessageId);
+
+  await handler(event);
+
+  const item = await s3().getObject({
+    Bucket: process.env.system_bucket,
+    Key: enabledQueueKey,
+  }).promise();
+
+  t.deepEqual(message, JSON.parse(JSON.parse(item.Body.toString())));
```

Comment:
Why do we have to do a double `JSON.parse`?

Reply:
I guess my question then is: does it have to be that way? Or is it only that way because of the code we've written?

Do we need to do a `JSON.stringify` on `message.Body` here?

https://github.com/nasa/cumulus/pull/2132/files?file-filters%5B%5D=.ts#diff-38eb9e3df7df0195f5bf20728f154f3d8d7ec21729d9884b010fae57d2e6c703R33

It seems like the message body returned from SQS should already be a string? 

https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_Message.html

So if it was JSON it would already have been stringified, meaning we don't need to stringify it again?